### PR TITLE
Issue #58 - Move doctrine auditing and enum logic to a bundle structure

### PR DESCRIPTION
**Pull request by kristofvc**

### CODE REVIEW

File: src/ExEss/Bundle/DoctrineExtensionsBundle/Schema/Trigger.php [hunk 1]
```diff
@@ -146,7 +146,6 @@ public function equals(Trigger $trigger): bool
             \preg_replace('/\s+/', '', $this->getStatement()) === \preg_replace('/\s+/', '', $trigger->getStatement())
             && $this->getTiming() === $trigger->getTiming()
             && $this->getOperation() === $trigger->getOperation()
-            // && $this->getTriggerName() === $trigger->getTriggerName();
```
Comment (bartdens) [inline src/ExEss/Bundle/DoctrineExtensionsBundle/Schema/Trigger.php:149]: why this change?

Comment (kristofvc) [reply src/ExEss/Bundle/DoctrineExtensionsBundle/Schema/Trigger.php:149]: that's commented code, I think it was because I said, during our pair programming session that $trigger->triggerName would work because it uses the same context, and we forgot to delete the using of methods afterwards

Comment (bartdens) [reply src/ExEss/Bundle/DoctrineExtensionsBundle/Schema/Trigger.php:149]: my bad, misinterpreted the diff completely wrong!

**Review by bartdens — APPROVED** [review]
